.github/workflows/feg-workflow.yml

```diff
@@ -0,0 +1,179 @@
+name: feg-workflow
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
can we add that: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths